unsplash-plugin-sample/server/README.md

```diff
@@ -0,0 +1,45 @@
+# Unsplash Images Plugin Sample 
+
+Follow these instructions to set up the server-side for the plugin. 
+## Install dependencies
+
+First ensure that your terminal is in the `plugin` folder of this project. To do this, use: 
+
+```bash
+cd /uxp-photoshop-plugin-samples/unsplash-plugin-sample/server
+```
+
+For `yarn` users, install all dependencies using:
+
+```
+yarn install
+```
+
+For `npm` users, install all dependencies using:
+
+```
+npm install
+```
+
+## Environment Variables Setup
+
+Next, you need to create a `.env` file containing all the variables. You can check the [.env.sample](./.env.sample) file for this. Enter all variables:
```

Comment:
Document what `PORT` does (i.e., something like "`PORT` specifies the port your local API server runs on")

Reply:
done